jobs/aws/eks-anywhere-build-tooling/cluster-api-provider-nutanix-presubmits.yaml

```diff
@@ -32,7 +32,7 @@ presubmits:
       automountServiceAccountToken: false
       containers:
       - name: build-container
-        image: public.ecr.aws/eks-distro-build-tooling/builder-base:91e37cccb386f038535ace5d9df762ec1750fc7b.2
+        image: public.ecr.aws/eks-distro-build-tooling/builder-base:latest
```

Comment:
We don't update this tag manually, there is automation around updating this and it already includes the 1.19 addition from last week.

Reply:
```
aws-eks-anywhere-prow-jobs git:(ntnx-latest-builder-base) docker run -it public.ecr.aws/eks-distro-build-tooling/builder-base:91e37cccb386f038535ace5d9df762ec1750fc7b.2 bash
Unable to find image 'public.ecr.aws/eks-distro-build-tooling/builder-base:91e37cccb386f038535ace5d9df762ec1750fc7b.2' locally
91e37cccb386f038535ace5d9df762ec1750fc7b.2: Pulling from eks-distro-build-tooling/builder-base
c9a5b97c2853: Already exists
4b3840ba3ffe: Already exists
adcbae174a96: Already exists
1e82624fb628: Already exists
e7688f18c177: Pull complete
ae60df8212ac: Pull complete
Digest: sha256:bc37082128294aab4b437909952f50cea4f1b1b31756f2f9e1a10189c485151e
Status: Downloaded newer image for public.ecr.aws/eks-distro-build-tooling/builder-base:91e37cccb386f038535ace5d9df762ec1750fc7b.2
bash-4.3# ls /go
bin  go1.13  go1.14  go1.15  go1.16  go1.17  go1.18  pkg  src
```

Reply:
In comparison, on `latest`
```
~ docker run -it public.ecr.aws/eks-distro-build-tooling/builder-base:latest bash
bash-4.3# ls /go
bin  go1.13  go1.14  go1.15  go1.16  go1.17  go1.18  go1.19  pkg  src
```

Reply:
my bad, just saw that #204 is still open. 1.19 should be on that.